src/index.tsx

```diff
@@ -548,7 +548,7 @@ class Cropper extends React.Component<CropperProps, State> {
               controls={false}
             >
               {(Array.isArray(video) ? video : [{ src: video }]).map((item) => (
-                <source {...item} />
+                <source key={item.type} {...item} />
```

Comment:
Type is missing for the default case where video is not an array.

Reply:
Switched to item.src as a key which should be unique.